hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -135,7 +137,12 @@ private void resolveMutableBuilderAttributes(
             @NonNull final Topic.Builder builder,
             @NonNull final Topic topic) {
         if (op.hasAdminKey()) {
-            builder.adminKey(op.adminKey());
+            var key = op.adminKey();
+            if (isEmpty(key)) {
+                builder.adminKey((Key) null);
```

Comment:
Why is this needed?

Reply:
If empty keyList - the adminKey is cleared. This is how to remove the adminKey.

Reply:
Ahh got it. I wasn't thinking this one through enough. Proceed!